powheg.sh

```diff
@@ -1,11 +1,11 @@
 package: POWHEG
 version: "%(tag_basename)s"
-tag: "r3178-alice1"
-source: https://github.com/alisw/POWHEG
+tag: "r3693-alice1"
+source: https://github.com/mfasDa/POWHEG
```

Comment:
I see you there as current code owner from your personal github.
How you want to proceed before the merge ? i.e. : 

1. should we leave that way for a while so that the chain is exercised to its end (cvmfs deployment... + execution test) while you still have the full hand on it ?
2. or do you want push alisw/POWHEG from your github to the central one before that merging this recipe to alisw/alidist ?

Note that I am fine personally with either option.

Reply:
Hi @maireiphc, as mentioned above ultimately the imported POWHEG version should appear as separate branch in alisw/POWHEG. As this is a 1-1 import of the POWHEG sources from the official svn repository it should normally be fine to import POWHEG directly into alisw/POWHEG and take the tag from there. 

I have already been running productions with the new POWHEG version built via this build recipe on lxplus.